mlrun/model.py

```diff
@@ -2072,6 +2074,17 @@ def __init__(
     def set_secrets(self, secrets):
         self._secrets = secrets
 
+    def _serialize_field(
+        self, struct: dict, field_name: str = None, strip: bool = False
+    ) -> typing.Any:
+        # We pull the field from self and not from struct because it was excluded from the struct when looping over
+        # the fields to save.
+        if field_name in ["start_time", "end_time"]:
+            value = getattr(self, field_name, None)
+            if value and isinstance(value, datetime):
+                return value.isoformat()
+        return super()._serialize_field(struct, field_name, strip)
```

Comment:
Can be simplified a bit.
```suggestion
        value = super()._serialize_field(struct, field_name, strip)
        # We pull the field from self and not from struct because it was excluded from the struct when looping over
        # the fields to save.
        if field_name in ("start_time", "end_time") and isinstance(value, datetime):
            return value.isoformat()
        return value
```

Reply:
nice, tnx!